bin/up-nobuild

```diff
@@ -8,6 +8,9 @@ set -e
 export DOCKER_IMAGE_TAG=${VERSION_TAG}
 
 if [ -z "$SKIP_IMAGE" ]; then
+  if [ ! -f "binaries/cf-operator-linux-amd64" ]; then
+    bin/build-linux-amd64
```

Comment:
This script is called 'nobuild', because it doesn't build the binary. So adding code here which does, is confusing :)

Actually this relates to what we talked about in the daily, we have one binary for two very different functions (server and utilities). That's why devs on OSX need a Linux binary in the Dockerimage, but a native one for running the server locally (see below line 20).

Anyhow, if that helps with OSX, we should add it, so devs can use `up-nobuild`. However I'd propose to put the OSX specific behaviour behind another check:

```
if [ `uname -s` = "Darwin" ]; then
  if [ ! -f "binaries/cf-operator-linux-amd64" ]; then
    bin/build-linux-amd64
  fi
fi
```